### PR TITLE
There is a lot of reflection warnings

### PR DESCRIPTION
**Pull request by vigneshsarma**
I have fixed the easy ones. There is some more left.

the remaining ones, 
```
Reflection warning, cider/nrepl/middleware/out.clj:53:22 - call to method write can't be resolved (target class is unknown).
Reflection warning, cider/nrepl/middleware/out.clj:55:24 - call to method write on java.lang.Object can't be resolved (no such method).
Reflection warning, cider/nrepl/middleware/out.clj:55:24 - call to method write on java.lang.Object can't be resolved (no such method).
Reflection warning, cider/nrepl/middleware/out.clj:57:22 - call to method write can't be resolved (target class is unknown).
Reflection warning, cider/nrepl/middleware/out.clj:59:24 - call to method write on java.lang.Object can't be resolved (no such method).
Boxed math warning, byte_streams/graph.clj:130:77 - call: public static double clojure.lang.Numbers.unchecked_add(java.lang.Object,double).
Reflection warning, cider/nrepl/middleware/out.clj:59:24 - call to method write on java.lang.Object can't be resolved (no such method).
Reflection warning, cider/nrepl/middleware/out.clj:61:21 - reference to field flush can't be resolved.
Reflection warning, cider/nrepl/middleware/out.clj:63:23 - reference to field flush on java.lang.Object can't be resolved.
Reflection warning, cider/nrepl/middleware/out.clj:63:23 - reference to field flush on java.lang.Object can't be resolved.

Reflection warning, cider/nrepl/middleware/inspect.clj:47:25 - call to java.lang.Exception ctor can't be resolved.

Reflection warning, cider/nrepl/middleware/debug.clj:159:29 - reference to field sym on java.lang.Object can't be resolved.
Reflection warning, cider/nrepl/middleware/debug.clj:159:29 - reference to field sym on java.lang.Object can't be resolved.
```

Not completly sure how to fix the remaining ones.

### CODE REVIEW

File: src/cider/nrepl/middleware/slurp.clj [hunk 1]
```diff
@@ -87,7 +87,7 @@
 (defn slurp-url-to-content+body
   "Attempts to parse and then to slurp a URL, producing a content-typed response."
   [url-str]
-  (if-let [url (try (URL. url-str)
+  (if-let [^URL url (try (URL. url-str)
                     (catch MalformedURLException e nil))]
```
Comment (cichli) [inline src/cider/nrepl/middleware/slurp.clj]: Could you fix the indentation here?

**Comment by cichli** [comment]
I have pushed a commit that will fix the Eastwood warnings, if you would like to rebase.

File: src/cider/nrepl/middleware/slurp.clj (hunk 1 above)
Comment (vigneshsarma) [reply src/cider/nrepl/middleware/slurp.clj]: Updated.

**Comment by vigneshsarma** [comment]
I have rebased it with master and also fixed the commit message.

I am not sure how to fix the remaining reflection warning. If you have ideas, I can fix them as well.

**Comment by cichli** [comment]
Fixed in d5f1f3a93b3378f8fdf79a223e4a1502b768c276. Thanks so much for looking at this – sorry it's taken me a while to get it merged.

>I am not sure how to fix the remaining reflection warning. If you have ideas, I can fix them as well.

If I get around to taking a look at them I'll ping you.

**Pull request closed without merging**